integrations/datadog/.port/resources/blueprints.json

```diff
@@ -0,0 +1,228 @@
+[
+  {
+    "identifier": "datadogHost",
+    "description": "This blueprint represents a datadog host",
+    "title": "Datadog Host",
+    "icon": "Datadog",
+    "schema": {
+      "properties": {
+        "up": {
+          "type": "boolean",
+          "title": "Is Running?",
+          "description": "Is the host up?"
+        },
+        "hostName": {
+          "type": "string",
+          "title": "Host Name",
+          "description": "the name of the host"
+        },
+        "description": {
+          "type": "string",
+          "title": "Description",
+          "description": "the project description"
```

Comment:
isn't there any relation that we can add to / from datadog host? isn't service is somehow connected to host? doesn't we have monitoring for hosts?

Reply:
I couldn't find any.